bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: set desired strimzi version

## Description
https://issues.redhat.com/browse/MGDSTRM-4198 - add desired strimzi version to kafka clusters

Left to DO:
- tests

## Verification Steps
Will be done later by tests. 

Alternatively: 
- run the service locally, create kafka and confirm that the value is set to the default one in the config or to one obtained from kasfleetshard, if the default value is an empty string

## Checklist (Definition of Done)
